projects/cdk/schematics/ng-update/constants/templates.ts

```diff
@@ -58,3 +70,33 @@ export const TAGS_TO_REPLACE: ReplacementTags[] = [
         addAttributes: ['tuiWrapper'],
     },
 ];
+
+export const ATTR_TO_DIRECTIVE: AttributeToDirective[] = [
+    {
+        componentSelector: 'tui-primitive-textfield',
+        inputProperty: '(autofilledChange)',
+        directive: '(tuiAutofilledChange)',
+        directiveModule: {
+            name: 'TuiAutofilledModule',
+            moduleSpecifier: '@taiga-ui/cdk',
+        },
+    },
+    {
+        componentSelector: TUI_INTERACTIVE_SELECTORS,
+        inputProperty: '(pressedChange)',
+        directive: '(tuiPressedChange)',
+        directiveModule: {
+            name: 'TuiPressedModule',
+            moduleSpecifier: '@taiga-ui/cdk',
+        },
+    },
+    {
+        componentSelector: TUI_INTERACTIVE_SELECTORS,
+        inputProperty: '(hoveredChange)',
+        directive: '(tuiHoveredChange)',
+        directiveModule: {
+            name: 'TuiHoveredModule',
+            moduleSpecifier: '@taiga-ui/cdk',
+        },
```

Comment:
I will move my lines to here in next PR 👌 
https://github.com/Tinkoff/taiga-ui/blob/6ba35d8000556291471e5626e83298a85dd00387/projects/cdk/schematics/ng-update/steps/migrate-sliders/migrate-input-slider.ts#L11-L21